shuffle.py

```diff
@@ -581,19 +569,17 @@ def populate(self):
         for (dirpath, dirnames, filenames) in os.walk(self.path):
             dirnames.sort()
             # Ignore the speakable directory and any hidden directories
-            if "ipod_control/speakable" not in dirpath.lower() and "/." not in dirpath.lower():
+            if "iPod_Control/Speakable" not in dirpath and "/." not in dirpath:
```

Comment:
What about this:
`"/." not in dirpath.lower():`
Do we need it here? I do not think so.